src/Servers/Kestrel/Transport.Abstractions/src/Internal/TransportConnection.FeatureCollection.cs

```diff
@@ -91,7 +92,7 @@ CancellationToken IConnectionLifetimeNotificationFeature.ConnectionClosedRequest
             set => ConnectionClosedRequested = value;
         }
 
-        void IConnectionLifetimeFeature.Abort() => Abort(abortReason: null);
+        void IConnectionLifetimeFeature.Abort() => Abort(new ConnectionAbortedException("The connection was aborted by the application via IConnectionLifetimeFeature.Abort()."));
```

Comment:
This exception string is used twice. Consider adding it to the resx.

Reply:
One is for ConnectionContext and the other is for IConnectionLifetimeFeature. ConnectionContext is defined in an assembly that doesn't currently have a resx which is probably the main reason it was left null before.